milli/src/index.rs

```diff
@@ -223,7 +223,7 @@ impl Index {
         let documents = env.create_database(&mut wtxn, Some(DOCUMENTS))?;
         wtxn.commit()?;
 
-        Index::set_creation_dates(&env, main, created_at, updated_at)?;
+        // Index::set_creation_dates(&env, main, created_at, updated_at)?;
```

Comment:
Is it OK to ignore creation dates for indexes going forward?

Reply:
Ho Gosh! This is the reason why the CI is failing. Thank you!
However, I disabled this because I was triggering a _deserialize error_ when using this tool on a database, and I think it was related to the way we compile the binaries (from the workspace or crate itself). I'll enable that back and see.